external/flume/src/test/scala/org/apache/spark/streaming/flume/FlumePollingStreamSuite.scala

```diff
@@ -53,6 +43,8 @@ class FlumePollingStreamSuite extends SparkFunSuite with BeforeAndAfter with Log
     .setMaster("local[2]")
     .setAppName(this.getClass.getSimpleName)
 
+  var utils = new PollingFlumeTestUtils
+
   def beforeFunction() {
     logInfo("Using manual clock")
     conf.set("spark.streaming.clock", "org.apache.spark.util.ManualClock")
```

Comment:
I think that can be moved into the `conf` above and before can be removed. Mind doing this small additional change?